packages/mdc-switch/mdc-switch.scss

```diff
@@ -192,3 +193,11 @@
     }
   }
 }
+
+.mdc-switch-label {
```

Comment:
Take this out of this PR. I want to discuss more if we want one `mdc-label` for switch, checkbox, and radio...or if we need three classes.